fvm/environment/block_info.go

```diff
@@ -146,6 +141,11 @@ func (info *blockInfo) GetBlockAtHeight(
 		return runtime.Block{}, false, nil
 	}
 
+	if info.blocks == nil {
+		return runtime.Block{}, false, errors.NewOperationNotSupportedError(
+			"GetBlockAtHeight")
+	}
+
```

Comment:
Move this check after the use of `info.blockHeader`, which is set in the remote debugger